staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -130,6 +130,11 @@ type KubeletConfiguration struct {
 	// Default: 10250
 	// +optional
 	Port int32 `json:"port,omitempty"`
+	// readOnlyBindAddress is the address for the Kubelet to serve on with
+	// no authentication/authorization. Defaults to the secure serving address if not set.
+	// Default: ""
```

Comment:
this is only used when `readOnlyPort` is non-zero, and defaults to use the value specified for `address`, right?

Reply:
Yes, that's the idea.